docs/docs/building-a-contact-form.md

```diff
@@ -43,6 +43,31 @@ Each method detailed below will start with the following contact form:
 
 ## Form submission options in Gatsby
 
+### Getform
+
+Getform is a form backend platform which offers a free-plan for handling form submissions on static sites. In order to begin receiving form submissions from your Gatsby site, you must first create a form on the platform directing the http POST method to the Getform, by placing the `name` attributes for the fields you want to make visible. (name, email, message etc.)
```

Comment:
```suggestion
Getform is a form backend platform which offers a free-plan for handling form submissions on static sites. In order to begin receiving form submissions from your Gatsby site, you must first create a form on the platform. When creating the form, direct the http POST method to the Getform, by placing the `name` attributes for the fields you want to make visible. (name, email, message etc.)
```
Which platform? Gatsby or Getform?

Reply:
Getform is the platform which can be used as a form submission handling option for Gatsby.

Reply:
Right. Just want to add some language to clarify this piece `you must first create a form on the platform`. It isn't clear which platform is being discussed.